### ISSUE TITLE
feat(model): seed transactional per-layer V4.1 decode state from prefill

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#13305]
# feat(model): seed transactional per-layer V4.1 decode state from prefill

```routing
repo: fak
lane: model
paths: ["internal/model/v41_forward.go","internal/model/v41_attention_state.go","internal/model/v41_decode_state_test.go"]
expected_steps: 3
```

## Working spine
A bounded V4.1 leaf toward 100/30; it does not establish either rate.

## Current state
v41ForwardState at v41_forward.go:121-131 holds history and one attention state, and the full layer loop resets attention state around :2066. V41AttentionState already has Prefill/Step and publication APIs (v41_attention_state.go:170-244). Decode currently recomputes all history.

## Why this is next
Bounds one native execution seam; throughput stays with the mission gate.

## Parent context
anthony-chaudhary/fak#12640; performance frontier anthony-chaudhary/fak#13294.

## Core through-line
Seed per-layer temporal state from full prefill: window KV and partial compression groups. Keep cross-layer publications in a step registry keyed by source identity and absolute compressed range. Stage bounded deltas plus original lengths/scalars; publish only after full prefill succeeds and roll back by truncating/restoring fields. Never deep-clone history-sized publications. Keep `Session.Step` on full history.

## Gold-plating boundary
State lifecycle only; no incremental arithmetic, no Step activation, no unbounded history-sized duplicate tensors.

## Worker packet
Target: DS4.1 or Qwen3.8. Own only listed paths. Three steps: independent regression; named seam; focused/package gates and land. Keep implementation/test contexts separate. If scope exceeds three implementation files, return a narrower successor.
Prerequisite plan leaf: 13. Resolve it to an issue ID in the dependency block before dispatch. Historical full-prefix behavior remains the reference until leaf 13 lands with an independent scalar-mask witness. Do not activate this state contract earlier.

## Done condition / witness
Witness: `go test ./internal/model -run 'TestV41DecodeState' -count=1`
The named test is a deliverable. Confirm `go test -list` includes it; empty match fails. Run owning-package and boundary gates after the last edit. Predeclare numerical tolerances.

## Verifiable Witness
```
go test ./internal/model -run 'TestV41DecodeState' -count=1
```

## Definition of done
- [ ] Tiny full prefill produces equivalent logits while seeding independent state for every layer; two sessions cannot alias mutable state.
- [ ] Injected later-layer failure leaves history, temporal state and the publication registry unchanged; rollback uses recorded lengths/scalars rather than a deep copy, and reset releases retained state.
- [ ] Allocation/copy counters remain constant as prefix length increases when retained bounded-window and partial-group sizes are fixed.
- [ ] Focused and owning-package gates pass after the final edit; independent reviewer reads the actual call path and resolving commit.

## Acceptance gate
All checklist items plus exact witness exit 0. Software validation is [SW-VERIFIED] only; physical throughput and coherent prompt-dependent generation stay with the physical qualification owner.

## Closure binding
Commit with DCO and this issue reference, explicit-path staging, and the (fak model) trailer; no agent-created PR.

## Likely files
- internal/model/v41_forward.go
- internal/model/v41_attention_state.go
- internal/model/v41_decode_state_test.go

## Expected steps
3

## Value
- Centrality: Core
- P1 Context: advanced - one source-backed execution seam per packet
- P2 Net value: preserved - parity, bounded memory and admission guards remain enforced
- P3 Adaptation: N/A - no adaptive policy in this leaf
- P4 Operations: advanced - exact worker scope and a witnessed production path

## Work estimate
Estimate: 3 points

## Overall completion contribution
Contribution: 3/8 points

## Completion standard
development

## Qualification status
No measured result; physical 100/30 acceptance remains outstanding.

## Dependencies
- Start blocked by: anthony-chaudhary/fak#13303

github_issue: anthony-chaudhary/fak#13305

**Comment by anthony-chaudhary** [comment]
Implementation and independent acceptance are complete in the local public checkout at commit \6c09650a5777b35b7154eb392eb6913fa893f925\; the commit is not yet published to \origin/main\, so this issue remains open. Witnesses: \go test ./internal/model -run '^TestV41DecodeState' -count=1\ passed 7/7; \go test ./internal/model/...\ and \go vet ./internal/model\ passed after the final rebase; exact-path \ak validate\ passed build, vet, and test; the canonical worker land completed admission, prospective/post-merge validation, and trunk CAS over exactly the three routed files. Independent review confirmed prefill parity, per-layer/session independence, late-layer rollback, reset release, source/range registry keys, and prefix-length-constant allocation counts. [SW-VERIFIED]; physical throughput remains outside this issue.
